packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -65,7 +65,16 @@
 			<section v-else class="no-open-tabs">
 				<img src="@assets/svg/seed.svg" alt="Seed" />
 				<p>You can open resources from the resource panel.</p>
-				<Button label="Open project overview" />
+				<Button
+					label="Open project overview"
+					@click="
+						openAsset({
+							assetName: 'Overview',
+							assetType: 'overview',
+							assetId: undefined
```

Comment:
It might be worth looking into making `assetId` optional.
Or at least `null` instead of `underfined` as you know its value.